README.md

```diff
@@ -75,7 +74,7 @@ This project supports below scenarios for end-to-end guest OS validation testing
 | Photon OS 3.x, 4.x                              | :heavy_check_mark:               | :heavy_check_mark:       | :heavy_check_mark:                 |
 | Ubuntu 18.04 and later live-server              | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
 | Ubuntu 20.04 and later cloud image              |                                  | :heavy_check_mark:       | :heavy_check_mark:                 |
-| Ubuntu 18.04 and later desktop                  |                                  |                          | :heavy_check_mark:                 |
+| Ubuntu 20.04 and later desktop                  | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
```

Comment:
How about "Ubuntu 18.04 desktop"?

Reply:
I have not tested ubuntu 18.04 this time. So exclude it.  We only need to test update release for ubuntu 20.04, 22.04 and new releases in the future.

Reply:
Thanks. Done